doc/configuration/network.md

```diff
@@ -80,6 +80,9 @@ rest:
   - `quarantine_duration` set the time to leave a node in quarantine before allowing
     it back (or not) into the fold.
     It is recommended to leave the default value `[default: 30min]`.
+  - `quarantine_whitelist` set a trusted list of peers that will not be quarantined in any circumstance. 
+    It should be a list of valid addresses, for example: `["/ip4/127.0.0.1"]`.
```

Comment:
I don't think it works that way. It should be a multi format address `["/ip4/127.0.0.1/tcp/3000"]` iirc

Reply:
It comes from the `MultiAddress` example. But I updated it so is a clearer example. Thanks!